f600233d1b154d507b782f84604bcb12/dictionary/x_g_dis_atat_acquisition_package.xml

```diff
@@ -27,35 +28,35 @@
             </choice>
         </element>
         <element label="Period Of Performance" max_length="32" name="period_of_performance" reference="x_g_dis_atat_period_of_performance" type="reference"/>
-        <element label="Primary Reviewer" max_length="32" name="primary_reviewer" reference="sys_user" type="reference"/>
+        <element label="DITCO CS" max_length="32" name="primary_reviewer" reference="x_g_dis_atat_ditco_contract_specialist" type="reference"/>
```

Comment:
I noticed in my PDI the `DITCO CS` column does not show up as expected but appears there. However, the column shows up in DSP. The under lying column is still named `primary_reviewer` is that intended?

Reply:
Yeah, unfortunately you can't change column names, just labels. 🙄

Reply:
Fair - I was considering leaving the label as-is too, but figured I'd compromise and change the label without actually destroying the whole field.